test/Microsoft.AspNetCore.Mvc.RazorPages.Test/Internal/ExecutorFactoryTest.cs

```diff
@@ -264,12 +265,34 @@ public class ExecutorFactoryTest
 
             // Assert
             Assert.NotNull(executor);
-            var actionResultTask = executor(new EmptyPage(), model);
+            var actionResultTask = executor(new EmptyPage(), model, GetArguments(methodInfo));
```

Comment:
You're correct that we don't need it for ALL these tests (I'll par it down to those that fail without it), but if we pass null we get NullRef's and if we pass the wrong number of args the executor throws an error. Do you think it would be better/cleaner to just manually construct those in the test or do you have another strategy in mind?